google-cloud-storage/lib/google/cloud/storage/project.rb

```diff
@@ -483,14 +485,26 @@ def create_bucket bucket_name, acl: nil, default_acl: nil,
         def signed_url bucket, path, method: nil, expires: nil,
                        content_type: nil, content_md5: nil, headers: nil,
                        issuer: nil, client_email: nil, signing_key: nil,
-                       private_key: nil, query: nil
-          signer = File::Signer.new bucket, path, service
-          signer.signed_url method: method, expires: expires, headers: headers,
-                            content_type: content_type,
-                            content_md5: content_md5,
-                            issuer: issuer, client_email: client_email,
-                            signing_key: signing_key, private_key: private_key,
-                            query: query
+                       private_key: nil, query: nil, version: :v2
+          case version
+          when :v2
+            signer = File::Signer.new bucket, path, service
```

Comment:
Would it make sense to rename `Signer` to `SignerV2` to be consistent with this new signer class?